TelcoColorCoderTest.cpp

```diff
@@ -0,0 +1,26 @@
+#include <iostream>
+#include <string>
+#include "assert.h"
+#include "TelcoColorCoderTest.h"
+#include "ColorCode.h"
+
+using namespace TelCoColorCoder;
+
+void testNumberToPair(int pairNumber,

```

Comment:
Good approach to separate tests into own file 👍

Reply:
Thank you